docs/docs/en/guide/datasource/cache.md

```diff
@@ -0,0 +1,23 @@
+# CACHE
+
+![cache](../../../../img/new_ui/dev/datasource/cache.png)
+
+## Datasource Parameters
+
+|       **Datasource**       |                                    **Description**                                    |
+|----------------------------|---------------------------------------------------------------------------------------|
+| Datasource                 | Select CACHE.                                                                         |
+| Datasource name            | Enter the name of the DataSource.                                                     |
+| Description                | Enter a description of the DataSource.                                                |
+| IP/Host Name               | Enter the CACHE service IP.(If there are multiple IPs, please separate them with `,`) |
+| Port                       | Enter the CACHE service port.                                                         |
+| Username                   | Set the username for CACHE connection.                                                |
+| Password                   | Set the password for CACHE connection.                                                |
+| Database name              | Enter the database name of the CACHE connection.                                      |
+| Jdbc connection parameters | Parameter settings for CACHE connection, in JSON format.                              |
+
+## Native Supported
+
+No, read section example in [datasource-setting](../installation/datasource-setting.md) `DataSource Center` section to activate
+this datasource.
+
```

Comment:
Please add more info about this datasource and put the official website in it.

Reply:
Sure, I'll get right on it.